datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/timeline/GetSchemaBlameResolver.java

```diff
@@ -0,0 +1,61 @@
+package com.linkedin.datahub.graphql.resolvers.timeline;
+
+import com.linkedin.common.urn.Urn;
+import com.linkedin.datahub.graphql.generated.GetSchemaBlameInput;
+import com.linkedin.datahub.graphql.generated.GetSchemaBlameResult;
+import com.linkedin.datahub.graphql.types.timeline.mappers.SchemaFieldBlameMapper;
+import com.linkedin.metadata.timeline.TimelineService;
+import com.linkedin.metadata.timeline.data.ChangeCategory;
+import com.linkedin.metadata.timeline.data.ChangeTransaction;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.net.URISyntaxException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import lombok.extern.slf4j.Slf4j;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.*;
+
+
```

Comment:
Minor: still lacking a resolver doc- can we add a comment detailing what this resolver does?

Reply:
Yes!